pyedr/MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 include ../README.rst ../LICENSE.txt
 include ../ChangeLog ../AUTHORS
 include setup.py setup.cfg
+include pyedr/tests/data/*
```

Comment:
Be more specific so that backup files are not included??? Not sure if it’s worth the effort but wanted to mention it.

Reply:
I'm not sure I understand what you're trying convey here - essentially the issue we have is that the datafiles needed to run the tests, using say `pytest --pyargs pyedr.tests` aren't being included in the package. This means that the tests won't run.

That being said, if we don't want to include the tests files here that's fine too, we just won't tell folks that they can test locally outside of a source installation. In some ways I would much prefer that because the tarball for `pyedr` with the test files is ~ 15 MB, compared to 11.8kB without.

Reply:
I just meant that instead of `*` use `*.xvg` or whatever more specific pattern includes the files.

Reply:
I see the point of not including test files with the package — seems overkill to make it 15 MB. 

If we exclude the files then the tests should produce a message that state how one can get them. 

I also don’t know how down-stream packagers like it if the tests cannot be run easily. 

Is there a way to include the files in the tar file but exclude from wheel and conda???

Reply:
Ah I see, in that case my vote would be towards it's not worth it. We shouldn't have backup files in there to begin with, and specific including file types means remembering to update the manifest every time we add a new file type (which can easily be forgotten about).

If we want to have the test files here, we should just bundle them all - that being said, I'm increasingly convinced that we shouldn't bundle them here, 14MB as a tarball means ~ 50 MB unpacked, that seems like too much of a burden, especially for what is likely to be an MDA core dependency.

Reply:
Oops hadn't seen your second message come in @orbeckst - :/ it _might_ be possible to make the tarball & wheels different, but then it's hard to make conda pick up something else that isn't the release or the pypi tarball (if that makes sense).

Essentially it gets rather messy.

One option here is to make `cat.edr` and `cat.xvg` much smaller given they are ~ 14M each (compared to ~ 100kB for the other files), @jbarnoud would need to weigh in on this here.

Reply:
If I recall correctly, I left the whole tests out initially to have the less cumbersome experience for the users. If pyedr is meant to be even leaner, it makes sense to have the same approach. I also like packages being light, I am currently rewriting some of my code in rust in part because shipping gigabytes of dependences was becoming impractical. 

I am not a huge fan of shipping the tests without the data though, as the data is an integral part of the tests. I should still be able to live with it.

Reply:
I tested using much smaller files in #55 and coverage seems unaffected, so if we want to reduce the test size I can get that PR ready for merging